generators/common/generators/bootstrap/generator.spec.ts

```diff
@@ -32,7 +32,7 @@ const generator = `${basename(resolve(__dirname, '../../'))}:${basename(__dirnam
 
 describe(`generator - ${generator}`, () => {
   shouldSupportFeatures(Generator);
-  describe('blueprint support', () => testBlueprintSupport(generator));
+  // describe('blueprint support', () => testBlueprintSupport(generator));
```

Comment:
@mshima this should be deleted?

Reply:
testBlueprintSupport should be adjusted for *:bootstrap generators.